core/executor/src/wasm_executor.rs

```diff
@@ -767,6 +768,42 @@ impl_function_executor!(this: FunctionExecutor<'e, E>,
 
 		Ok(offset)
 	},
+	ext_local_network_state(msg_len: *mut u32) -> *mut u8 => {
+		let res = this.ext.offchain()
+			.map(|api| api.local_network_state())
+			.ok_or_else(|| "Calling unavailable API ext_local_network_state: wasm")?;
+
+		let encoded = res.encode();
+		let len = encoded.len() as u32;
+		let offset = this.heap.allocate(len)? as u32;
+		this.memory.set(offset, &encoded)
+			.map_err(|_| "Invalid attempt to set memory in ext_local_network_state")?;
+
+		this.memory.write_primitive(msg_len, len)
+			.map_err(|_| "Invalid attempt to write msg_len in ext_local_network_state")?;
+
+		Ok(offset)
+	},
+	ext_local_authority_pubkey(
+		kind: u32,
+		written_out: *mut u32
+	) -> *mut u8 => {
+		let kind = offchain::CryptoKind::try_from(kind)
+			.map_err(|_| "crypto kind OOB while ext_decrypt: wasm")?;
```

Comment:
```suggestion
			.map_err(|_| "crypto kind OOB while ext_local_authority_pubkey: wasm")?;
```

Reply:
Wait, is it acually OOB?

Reply:
Yeah, I checked again. Also I just copied this line from the analog methods.